app/models/mixins/supports_feature_mixin.rb

```diff
@@ -58,38 +58,40 @@ module SupportsFeatureMixin
-    :disassociate_floating_ip     => 'Disassociate a Floating IP',
-    :discovery                    => 'Discovery of Managers for a Provider',
-    :evacuate                     => 'Evacuation',
-    :events                       => 'Query for events',
-    :launch_cockpit               => 'Launch Cockpit UI',
-    :live_migrate                 => 'Live Migration',
-    :migrate                      => 'Migration',
-    :provisioning                 => 'Provisioning',
-    :reboot_guest                 => 'Reboot Guest Operation',
-    :reconfigure                  => 'Reconfiguration',
-    :refresh_new_target           => 'Refresh non-existing record',
-    :regions                      => 'Regions of a Provider',
-    :remove_host                  => 'Remove Host',
-    :resize                       => 'Resizing',
-    :retire                       => 'Retirement',
-    :smartstate_analysis          => 'Smartstate Analaysis',
-    :snapshots                    => 'Snapshots',
-    :terminate                    => 'Terminate a VM',
-    :update_aggregate             => 'Host Aggregate Update',
-    :refresh_network_interfaces   => 'Refresh Network Interfaces for a Host',
+    :control                    => 'Basic control operations',
+    :cloud_tenant_mapping       => 'CloudTenant mapping',
+    :create                     => 'Creation',
+    :backup_create              => 'CloudVolume backup creation',
+    :backup_restore             => 'CloudVolume backup restore',
+    :cinder_service             => 'Cinder storage service',
+    :create_host_aggregate      => 'Host Aggregate Creation',
+    :swift_service              => 'Swift storage service',
+    :delete                     => 'Deletion',
+    :delete_aggregate           => 'Host Aggregate Deletion',
+    :disassociate_floating_ip   => 'Disassociate a Floating IP',
+    :discovery                  => 'Discovery of Managers for a Provider',
+    :evacuate                   => 'Evacuation',
+    :events                     => 'Query for events',
+    :launch_cockpit             => 'Launch Cockpit UI',
+    :live_migrate               => 'Live Migration',
+    :migrate                    => 'Migration',
+    :provisioning               => 'Provisioning',
+    :reboot_guest               => 'Reboot Guest Operation',
+    :reconfigure                => 'Reconfiguration',
+    :refresh_new_target         => 'Refresh non-existing record',
+    :regions                    => 'Regions of a Provider',
+    :remove_host                => 'Remove Host',
+    :resize                     => 'Resizing',
+    :retire                     => 'Retirement',
+    :smartstate_analysis        => 'Smartstate Analaysis',
+    :snapshots                  => 'Snapshots',
+    :terminate                  => 'Terminate a VM',
+    :update_aggregate           => 'Host Aggregate Update',
+    :refresh_network_interfaces => 'Refresh Network Interfaces for a Host',
+    :update                     => 'Update',
```

Comment:
@blomquisg shouldn't this be something more specific like `:cloud_volume_snapshot_update` or simply `:snapshot_update` ?
/cc @durandom @tzumainn @petrblaho